lib/routes/account.js

```diff
@@ -1568,60 +1472,12 @@ module.exports = function (
   if (config.isProduction) {
     delete routes[0].config.validate.payload.preVerified
   } else {
-    // programmatic account lockout is only available in non-production mode.
```

Comment:
nit: it may be worth keeping the comment here, with s/is/was/, to explain the otherwise-mysterious branching logic.